### PR TITLE
GH-12222: Use OrderPlacedEvent from @spartacus/checkout/root instead.

### LINKED ISSUE #12222: Dependency cleanup -> [cds] -> OrderPlacedEvent
It's fine that CDS depends on checkout.

- change the import path to use the event in checkout root.

May be resolved in part by GH-12185

```
integration-libs/cds/src/profiletag/services/profile-tag-push-events.service.ts:8:3 - error TS2305: Module '"@spartacus/core"' has no exported member 'OrderPlacedEvent'.
8   OrderPlacedEvent,
```

### PR DESCRIPTION
**Pull request by plabadie**
closes: GH-12222

### CODE REVIEW

File: integration-libs/cds/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "@angular/core": "^10.1.0",
     "@angular/router": "^10.1.0",
     "@ngrx/store": "^10.0.0",
+    "@spartacus/checkout": "3.0.0",
```
Comment (WeizhengSap) [inline integration-libs/cds/package.json]: the version is 3.0.0 or 3.2.0?

Comment (plabadie) [reply integration-libs/cds/package.json]: Latest release is 3.3.0.  I'll align on that as a baseline.  In the end, checkout lib should be first released as 4.0

**Review by WeizhengSap — APPROVED** [review]

File: integration-libs/cds/package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "@angular/core": "^10.1.0",
     "@angular/router": "^10.1.0",
     "@ngrx/store": "^10.0.0",
+    "@spartacus/checkout": "3.3.0",
```
Comment (WeizhengSap) [inline integration-libs/cds/package.json]: also need to add it to `.release-it.json` and `ng-package.json`

File: integration-libs/cds/package.json (hunk 1 above)
Comment (plabadie) [reply integration-libs/cds/package.json]: Actually I'll use `3.2.0-next.2`. This version on develop is intentional.

File: integration-libs/cds/package.json (hunk 2 above)
Comment (plabadie) [reply integration-libs/cds/package.json]: Thanks, good catch.

**Pull request merged**